tests/installer.spec.ts

```diff
@@ -55,7 +55,7 @@ describe('getInstallerInfo', () => {
         'platform': 'macos',
         'type': 'dmg',
         'arch': 'universal',
-        'url': 'https://d3gcli72yxqn2z.cloudfront.net/downloads/desktop/macos/1.2.0/stable/universal/ibm-aspera-browser_1.2.0_macos.dmg'
+        'url': 'https://d3gcli72yxqn2z.cloudfront.net/downloads/desktop/macos/1.2.0/stable/universal/ibm-aspera-sdk_1.2.0_macos.dmg'
```

Comment:
Those links are wrong... probably some bad search and replace

Reply:
No. They are just fake links for unit tests.